aws/table_aws_wellarchitected_check_detail.go

```diff
@@ -0,0 +1,311 @@
+package aws
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedCheckDetail(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_check_detail",
+		Description: "AWS Well-Architected Check Detail",
+		List: &plugin.ListConfig{
+			ParentHydrate: listWellArchitectedWorkloads,
+			Hydrate:       listWellArchitectedCheckDetails,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "workload_id", Require: plugin.Optional},
+				{Name: "choice_id", Require: plugin.Optional},
+				{Name: "lens_arn", Require: plugin.Optional},
+				{Name: "pillar_id", Require: plugin.Optional},
+				{Name: "question_id", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(wellarchitectedv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "id",
+				Description: "Trusted Advisor check ID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CheckDetail.Id"),
+			},
+			{
+				Name:        "choice_id",
+				Description: "The ID of a choice.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CheckDetail.ChoiceId"),
+			},
+			{
+				Name:        "description",
+				Description: "Trusted Advisor check description.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CheckDetail.Description"),
+			},
+			{
+				Name:        "flagged_resources",
+				Description: "Count of flagged resources associated to the check.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("CheckDetail.FlaggedResources"),
+			},
+			{
+				Name:        "lens_arn",
+				Description: "Well-Architected Lens ARN associated to the check.",
```

Comment:
```suggestion
				Description: "The Amazon Resource Name (ARN) of the lens.",
```

Would you kindly provide an update to the description in order to standardize the ARN (Amazon Resource Name) description?